.github/workflows/examples-nrfconnect.yaml

```diff
@@ -192,6 +192,41 @@ jobs:
                     nrfconnect nrf5340dk_nrf5340_cpuapp lighting-app \
                     examples/lighting-app/nrfconnect/build/zephyr/zephyr.elf \
                     /tmp/bloat_reports/
+            - name: Build example nRF Connect SDK Lock App on nRF7002 PDK
+              if: github.event_name == 'push' || steps.changed_paths.outputs.nrfconnect == 'true'
+              timeout-minutes: 20
```

Comment:
will the total build timeout of 140 minutes (line 28 above) still be sufficient after adding these 4 extra compile examples? Checked one latest build like https://github.com/project-chip/connectedhomeip/actions/runs/3541691183/jobs/5946278237 and it looks we use 98 minutes before these builds.

Reply:
https://github.com/project-chip/connectedhomeip/actions/runs/3524523931/jobs/5910046550 for this PR finished a lot faster though ... so maybe we are fine. On average each app seems to be 5-7 minutes instead of 20.